pkgs/applications/networking/warp/default.nix

```
@@ -15,6 +15,8 @@
 , glib
 , gtk4
 , libadwaita
+, Security
+, Foundation
```

```suggestion
, darwin
```

For the sake of RFC 140?

---

Depending on the singular deps rather than the package set done everywhere else in Nixpkgs and closer to what we do for regular `pkgs` deps, so I won't stray away from that.

Once we (as a broader community, not necessarily through an RFC) decide which way to go, we can migrate all of these in one go should we need to. I have no particularly strong preference either way.